web-app/src/main/webapp/script/loadData.js

```diff
@@ -82,7 +83,7 @@ function createMarkers(map, reports) {
           lng: Number(report.longitude),
         },
         map: map,
-        date: report.yearMonth,
+        date: new Date(report.timestamp*1000),
```

Comment:
nit: use the const reportsDate here?

Reply:
Oh actually fun thing, reportsDate is not in the same scope because it is in a different function. Github cuts bunch of code in the middle and the indentation happened to make it seem like it belongs to same function as filterReports :(